src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/ScannableItem.java

```diff
@@ -74,6 +77,7 @@ public ScannableItem(
         this.ocrData = ocrData;
         this.fileName = fileName;
         this.notes = notes;
+        this.documentType = (documentType == null ? "Other" : documentType);
```

Comment:
Document type should not be null and if it is null contract is broken. There will be validation in place even before it is passed to data layer

Reply:
I started from the assumption that we still accept inputs using the "old" model, which does not have the new fields. This means accepting nulls and converting them to the default value. If not the case this can be changed. Please be mindful that "being permissive in what you accept and strict in what you send" is a standard principle while building an api.